src/components/Datatable/Datatable.tsx

```diff
@@ -49,7 +49,11 @@ function Datatable<D extends Record<string, unknown>>({
   isBatchModuleEnabled = true,
   controlsConfig = {},
   tableConfig = {},
-}: DatatableProps<D>): React.ReactElement {
+  pageButtonsCount,
+  shouldCloseActionsDropdownOnItemClick,
```

Comment:
@julesbarr I think we can actually drop this property and close actions whenever the action is clicked. I don't see many reasons why anyone would want to keep it open after click and dismissing the menu is default action anyway on click